### DISCUSSION TITLE
save html with identify function

### DISCUSSION THREAD

**Discussion by AliceHuangn** [opengeos/leafmap#242] [Q&A] [answered]
Hi Dr. Wu, 

I saved my map in html, and there is no identify function of the html. How can I save it with the identify function? Thanks

**Comment by giswqs** [comment, marked as answer]
Unfortunately, the exported HTML won't support bidirectional functionality. That's all custom tool widgets won't show up in the exported HTML. That's the limitation of folium and ipyleaflet.
